webkit2/webkit2.find-controller.lisp

```diff
@@ -15,11 +15,54 @@
    ("text" "gchararray")
    ("web-view" "WebKitWebView" t t)))
 
-(defcfun "webkit_find_controller_search" :void
+(define-g-enum "WebKitFindOptions" webkit-find-options ()
+  :webkit-find-options-none
+  (:webkit-find-options-case-insensitive 1)
+  (:webkit-find-options-at-word-starts 2)
+  (:webkit-find-options-treat-medial-capital-as-word-start 4)
+  (:webkit-find-options-backwards 8)
+  (:webkit-find-options-wrap-around 16))
+
+(defcfun ("webkit_find_controller_search" %webkit-find-controller-search) :void
   (find-controller (g-object webkit-find-controller))
   (search-text :string)
   (find-options :uint)
   (max-match-count :uint))
+
+(defun webkit-find-controller-search (controller search-text
+                                      &key (max-match-count
+                                            ;; I.e., UINT_MAX
+                                            (1- (expt 2 (* 8 (cffi:foreign-type-size :uint)))))
```

Comment:
You can use `#.` to exapnd this at read-time and save a small computation :)

EDIT: Probably a bad idea, as (cffi:foreign-type-size :uint) is architecture-dependent and must be run on the host machine.